test/e2e/ha/scale_core_contrail_services_test.go

```diff
@@ -0,0 +1,338 @@
+			Spec: contrail.ProvisionManagerSpec{
+				CommonConfiguration: contrail.CommonConfiguration{
+					Create:       &trueVal,
+					NodeSelector: map[string]string{"node-role.kubernetes.io/master": ""},
+				},
+				ServiceConfiguration: contrail.ProvisionManagerConfiguration{
+					Containers: []*contrail.Container{
+						{Name: "init", Image: "registry:5000/common-docker-third-party/contrail/python:" + versionMap["python"]},
+						{Name: "provisioner", Image: "registry:5000/contrail-operator/engprod-269421/contrail-operator-provisioner:" + versionMap["contrail-operator-provisioner"]},
+					},
+				},
+			},
+		}
+
+		cluster := &contrail.Manager{
+			ObjectMeta: meta.ObjectMeta{
+				Name:      "cluster1",
+				Namespace: namespace,
+			},
+			Spec: contrail.ManagerSpec{
+				CommonConfiguration: contrail.CommonConfiguration{
+					HostNetwork: &trueVal,
+				},
+				Services: contrail.Services{
+					Cassandras:       cassandras,
+					Zookeepers:       zookeepers,
+					Config:           config,
+					Webui:            webui,
+					Rabbitmq:         rabbitmq,
+					ProvisionManager: provisionManager,
+				},
+			},
+		}
+
+		t.Run("when manager resource with Config and dependencies are created", func(t *testing.T) {
+			t.Skip()
+			var replicas int32 = 1
+			_, err := controllerutil.CreateOrUpdate(context.Background(), f.Client.Client, cluster, func() error {
+				cluster.Spec.CommonConfiguration.Replicas = &replicas
+				return nil
+			})
+
+			assert.NoError(t, err)
+
+			w := wait.Wait{
+				Namespace:     namespace,
+				Timeout:       waitTimeout,
+				RetryInterval: retryInterval,
+				KubeClient:    f.KubeClient,
+				Logger:        log,
+			}
+
+			t.Run("then a ready Zookeeper StatefulSet should be created", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-zookeeper-zookeeper-statefulset", replicas))
+			})
+
+			t.Run("then a ready Cassandra StatefulSet should be created", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-cassandra-cassandra-statefulset", replicas))
+			})
+
+			t.Run("then a ready Config StatefulSet should be created", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-config-config-statefulset", replicas))
+			})
+
+			t.Run("then a ready webui StatefulSet should be created", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-webui-webui-statefulset", replicas))
+			})
+
+			t.Run("then a ready provisionmanager StatefulSet should be created", func(t *testing.T) {
+				assert.NoError(t, w.ForReadyStatefulSet("hatest-provmanager-provisionmanager-statefulset", replicas))
+			})
+		})
+
+		t.Run("when replicas is set to 3 in manager", func(t *testing.T) {
+			var replicas int32 = 3
+			_, err := controllerutil.CreateOrUpdate(context.Background(), f.Client.Client, cluster, func() error {
+				cluster.Spec.CommonConfiguration.Replicas = &replicas
+				return nil
+			})
+
+			assert.NoError(t, err)
+
+			w := wait.Wait{
```

Comment:
`w := wait.Wait` can be extracted from these t.Runs and defined once

Reply:
Done